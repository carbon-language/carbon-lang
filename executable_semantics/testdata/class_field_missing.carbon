// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
// Exceptions. See /LICENSE for license information.
// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception

package ExecutableSemanticsTest api;

class Point {
  var x: i32;
  var y: i32;
}

fn main() -> i32 {
  return Point(.x = 1).x - 1;
}
