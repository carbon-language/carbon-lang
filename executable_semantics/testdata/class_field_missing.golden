COMPILATION ERROR: executable_semantics/testdata/class_field_missing.carbon:11: type error in call
expected: (x = i32, y = i32)
actual: (x = i32)
EXIT CODE: 255
